lib/tasks/git-init.js

```diff
@@ -45,6 +50,20 @@ class GitInitTask extends Task {
     return execa('git', ['--version']);
   }
 
+  _gitEmailConfigured() {
+    if (process.env.GIT_COMMITTER_EMAIL) {
+      return Promise.resolve();
```

Comment:
we might want to resolve with `process.env.GIT_COMMITTER_EMAIL`; it's always confusing to consuming promises with empty fulfilled value.